src/Symfony/Component/Workflow/MarkingStore/MultipleStateMarkingStore.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Symfony\Component\Workflow\MarkingStore;
 
+@trigger_error(sprintf('"%s" is deprecated since Symfony 4.3. Use "%s" instead.', MultipleStateMarkingStore::class, MethodMarkingStore::class), E_USER_DEPRECATED);
```

Comment:
`s/Symfony 4.3. Use/Symfony 4.3, use/` (same in the annotation and in other files below)